android_jni/avifandroidjni/src/main/jni/CMakeLists.txt

```diff
@@ -35,6 +35,8 @@ add_subdirectory(../../../../.. build)
 
 add_library("avif_android" SHARED "libavif_jni.cc")
 
+target_compile_options(avif_android PRIVATE $<TARGET_PROPERTY:avif,COMPILE_OPTIONS>)
```

Comment:
Vignesh: I didn't review the change to this file. The changes to libavif_jni.cc seem like a good idea.

Reply:
This change isn't actually necessary and I'd be fine with removing it. It makes it so that this warning causes errors on build, but that's not strictly necessary. If something like #1778 gets merged then this would be redundant.